docs/install/virtual_machine.md

```diff
@@ -175,10 +178,6 @@ plt.title('How fast do you want to go today?')
 show(plt)
 ``` 
 
+### R Extras
```

Comment:
The issue is how github markdown works vs the doc generator.
Github (readme) markdown uses the name of the header, in lower case, with spaces converted to dashes.
Is there an example of how the doc md -> html compiler interprets anchors?  I looked around at all of the other zeppelin docs, and couldn't find any doc examples that used anchors to headers within the "current" document.  I can update for the docs once I confirm how the doc generator works.  Does it use `id`?  Any examples?